src/components/accessories/laboratory/filter/ExamFilterForm.tsx

```diff
@@ -0,0 +1,225 @@
+                    fieldValue={formik.values.dateFrom}
+                    disableFuture={false}
+                    format="dd/MM/yyyy"
+                    isValid={isValid("dateFrom")}
+                    errorText={getErrorText("dateFrom")}
+                    label={t("lab.fromdate")}
+                    onChange={dateFieldHandleOnChange("dateFrom")}
+                  />
+                </div>
+                <div className="filterLabForm__item">
+                  <DateField
+                    fieldName="dateTo"
+                    fieldValue={formik.values.dateTo}
+                    disableFuture={false}
+                    theme="regular"
+                    format="dd/MM/yyyy"
+                    isValid={isValid("dateTo")}
+                    errorText={getErrorText("dateTo")}
+                    label={t("lab.todate")}
+                    onChange={dateFieldHandleOnChange("dateTo")}
+                  />
+                </div>
+
+                <div className="filterLabForm__item">
+                  <DateField
+                    fieldName="month"
+                    views={["month"]}
+                    fieldValue={formik.values.month}
+                    disableFuture={true}
+                    theme="regular"
+                    format="MMMM"
+                    isValid={isValid("month")}
+                    errorText={getErrorText("month")}
+                    label={t("lab.month")}
+                    onChange={dateFieldHandleOnChange("month")}
+                  />
+                </div>
+                <div className="filterLabForm__item">
+                  <DateField
+                    fieldName="year"
+                    views={["year"]}
+                    fieldValue={formik.values.year}
+                    disableFuture={true}
+                    theme="regular"
+                    format="yyyy"
+                    isValid={isValid("year")}
+                    errorText={getErrorText("year")}
+                    label={t("lab.year")}
+                    onChange={dateFieldHandleOnChange("year")}
+                  />
+                </div>
+                <div className="filterLabForm__item">
+                  <AutocompleteField
+                    fieldName="examName"
+                    fieldValue={formik.values.examName}
+                    label={t("lab.exam")}
+                    isValid={isValid("examName")}
+                    errorText={getErrorText("examName")}
+                    onBlur={onBlurCallback("examName")}
+                    options={examOptions}
+                  />
+                </div>
+              </div>
+            </div>
+            <div className="filterForm__buttonSet">
+              <Button variant="contained" type="submit">
+                {t("lab.filter")}
+              </Button>
+            </div>
+          </form>
+        </AccordionDetails>
+      </Accordion>
+    </div>
+  );
+};
```

Comment:
![image](https://user-images.githubusercontent.com/8697550/177333883-392bc625-c856-421c-a1f7-af7f5860985a.png)

THe filter form should have the same width of the rest of the elements.